tests/galsim_test_helpers.py

```diff
@@ -361,7 +361,8 @@ def all_obj_diff(objs):
         for j, objj in enumerate(objs):
             if i == j:
                 continue
-            assert obji != objj, "Found equivalent objects {} == {}".format(obji, objj)
+            assert obji != objj, ("Found equivalent objects {} == {} at indices {} and {}"
+                                  .format(obji, objj, i, j))
```

Comment:
Python 2.7.0 was released July 3rd, 2010, so it's even older than @rmandelb 's ancient previous installation of scipy.  I vote drop support for 2.6.